ibmcloud/image/build.sh

```diff
@@ -185,7 +235,7 @@ case "$(uname -m)" in
     s390x)
         ;;
     *)
-        echo -e "\nUpdating initramfs\n"
+        echo -e "\nUpdating initramfs"
```

Comment:
minor: `printf` could be used here. As this is more style, you might do a follow-up PR.

Reply:
fixed.